app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
 class ApplicationController < ActionController::Base
+  protected
+
+  def after_sign_in_path_for(resource)
+    # return the path based on resource
```

Comment:
i think we will need this for assigning different roles to admin/user model - it was mentioned in the tutorial as needed for that but we haven't gotten to that part yet so im not quite sure; if we end up not needing it we can get rid of it